common/on_action/game_start.txt

```diff
@@ -160,6 +160,8 @@ on_game_start_after_lobby = {
 		# Warcraft
 		WCRAC.1		#Assigns race traits at the game start
 
+		WCRAC.6		#Gives race modifier for army
```

Comment:
The game gives modifiers at the start, so new characters don't have them.
Why you didn't make them innovations?